attractions/dropdown/dropdown-shell.svelte

```diff
@@ -49,6 +55,15 @@
       : document.removeEventListener('keydown', handleKeyPress));
 
   const dispatch = createEventDispatcher();
+
+  const isDropdownOpen = writable(open);
+  const getDropdownShellBoundary = writable(() => {});
```

Comment:
Perhaps this would suffice?
```suggestion
  function getDropdownShellBoundary() {
    if (self == null) {
      return null;
    }
    
    return self.getBoundingClientRect();
  }
```

Reply:
Yup I'm overthinking it LOL. I make your suggestion shorter:

```typescript
const getDropdownShellBoundary = () => self?.getBoundingClientRect();
```